packages/blockchain/src/blockchain.ts

```diff
@@ -1322,7 +1322,7 @@ export class Blockchain implements BlockchainInterface {
       number: 0,
       stateRoot,
       withdrawalsRoot: common.isActivatedEIP(4895) ? KECCAK256_RLP : undefined,
-      excessDataGas: common.isActivatedEIP(4844) ? BigInt(0) : undefined,
+      excessDataGas: common.isActivatedEIP(4844) ? common.genesis().excessDataGas : undefined,
```

Comment:
TODO (check): I think this line is unnecessary, it takes params from `common.genesis()` first and copies those?